common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/ElasticRetriever.scala

```diff
@@ -40,17 +40,19 @@ class ElasticRetriever[T](client: ElasticClient, index: Index)(
           // original IDs.
           // See https://www.elastic.co/guide/en/elasticsearch/reference/6.8/docs-multi-get.html
           val documents = result.docs
-            .map { _.safeTo[T] }
             .zip(ids)
```

Comment:
Are we sure these always return in the order requested?

Reply:
GitHub has clipped the comment; yes:

```
// Documents are guaranteed to be returned in the same order as the
// original IDs.
// See https://www.elastic.co/guide/en/elasticsearch/reference/6.8/docs-multi-get.html
```